cmake/modules/configuration_files.cmake

```diff
@@ -118,3 +118,4 @@ zephyr_boilerplate_watch(DTC_OVERLAY_FILE)
 
 zephyr_get(EXTRA_CONF_FILE SYSBUILD LOCAL VAR EXTRA_CONF_FILE OVERLAY_CONFIG MERGE REVERSE)
 zephyr_get(EXTRA_DTC_OVERLAY_FILE SYSBUILD LOCAL MERGE REVERSE)
+zephyr_get(DTS_EXTRA_CPPFLAGS SYSBUILD LOCAL MERGE REVERSE)
```

Comment:
The problem I see with this @tejlmand is this is now an ever expanding list of additional fields that are being used, can't this be shared somehow with the underlying zephyr code so that there is a common file of what goes in here? Not blocking comment